### PR TITLE
Implemented /users route

### PR DESCRIPTION
**Pull request by FabianNowak**

### CODE REVIEW

File: Backend/src/database/services/user.service.ts [hunk 1]
```diff
@@ -18,6 +18,29 @@ export default class UserService {
     return await userModel.find().exec();
   }
 
+  static async searchUsers(keyword: string | null | undefined, sortBy: string | null | undefined) {
```
Comment (Pellumbengineer) [inline Backend/src/database/services/user.service.ts:21]: You can add comments for this searchUsers Method

File: Backend/src/database/services/user.service.ts [hunk 2]
```diff
@@ -18,6 +18,29 @@ export default class UserService {
...
+    let sort = 1;
+    if (sortBy == "namedesc") {
+      sort = -1;
+    }
+    let keywordFilter: FilterQuery<IUser> = {};
+    if (keyword && keyword.length > 0) {
+      keywordFilter = { $or: [{ firstName: keyword }, { lastName: keyword }] };
+    }
+    return await userModel
+      .find(keywordFilter, {
```
Comment (Pellumbengineer) [inline Backend/src/database/services/user.service.ts:32]: Are you only returning the id of the found user here?

Comment (FabianNowak) [reply Backend/src/database/services/user.service.ts:32]: Only non-private info like Name avatar and note. Excluding unnecessary/private info.

**Review by Pellumbengineer — APPROVED** [review]
LGTM

**Pull request merged**